packages/stark-build/config/webpack.dev.js

```diff
@@ -36,296 +36,298 @@ module.exports = function (env) {
-			launch: MONITOR, // -> default 'false'
-			port: 3030, // default -> 8081
-			excludeSourceMaps: true // default 'true'
-		})
-	],
-
-	/**
-	 * Webpack Development Server configuration
-	 * Description: The webpack-dev-server is a little node.js Express server.
-	 * The server emits information about the compilation state to the client,
-	 * which reacts to those events.
-	 *
-	 * See: https://webpack.github.io/docs/webpack-dev-server.html
-	 */
-	devServer: {
-		port: METADATA.PORT,
-		host: METADATA.HOST,
-		hot: METADATA.HMR,
-		inline: true,
-		compress: true,
-		overlay: {
-			errors: false,
-			warnings: false
-		},
-		// public: METADATA.PUBLIC, // TODO check if needed/useful in our case?
-
-		// HTML5 History API support: no need for # in URLs
-		// automatically redirect 404 errors to the index.html page
-		// uses connect-history-api-fallback behind the scenes: https://github.com/bripkens/connect-history-api-fallback
-		// reference: http://jaketrent.com/post/pushstate-webpack-dev-server/
-		historyApiFallback: true,
-
-		// Due to a security fix of Webpack Server, we enable this option to permit the access to the application when running on 0.0.0.0
-		// reference: https://github.com/webpack/webpack-dev-server/releases/tag/v2.4.3
-		disableHostCheck: true,
-
-		// file watch configuration
-		// reference: https://webpack.js.org/configuration/watch/
-		watchOptions: {
-			// if you're using Docker you may need this
-			aggregateTimeout: 300,
-			poll: 1000
-			// ignored: /node_modules/ // node_modules should also be watched for any changes in @nationalbankbelgium
-		},
-		// TODO: to enable this we should make Webpack to write dist files to disk
-		// contentBase: helpers.root(buildUtils.getApplicationAssetsConfig().outDir), // necessary so that all the content is available (e.g., app assets, stark assets, ...)
+			/**
+			 * Plugin: WebpackMonitor
+			 * Description: This plugins give us information about the bundle size
+			 * See: https://github.com/webpackmonitor/webpackmonitor
+			 */
+			new WebpackMonitor({
+				capture: MONITOR, // -> default 'true'
+				target: helpers.root("reports/webpack-monitor/stats.json"), // default -> '../monitor/stats.json'
+				launch: MONITOR, // -> default 'false'
+				port: 3030, // default -> 8081
+				excludeSourceMaps: true // default 'true'
+			})
+		],
 
 		/**
-		 * Here you can access the Express app object and add your own custom middleware to it.
+		 * Webpack Development Server configuration
+		 * Description: The webpack-dev-server is a little node.js Express server.
+		 * The server emits information about the compilation state to the client,
+		 * which reacts to those events.
 		 *
 		 * See: https://webpack.github.io/docs/webpack-dev-server.html
 		 */
-		before: function (app) {
-			// For example, to define custom handlers for some paths:
-			// app.get('/some/path', function(req, res) {
-			//   res.json({ custom: 'response' });
-			// });
-		},
+		devServer: {
```

Comment:
All the styling changes are really annoying in our PRs. I think we should isolate those in separate PRs if possible.